internal/sdkv2provider/schema_cloudflare_access_policy.go

```diff
@@ -12,32 +12,38 @@ import (
 func resourceCloudflareAccessPolicySchema() map[string]*schema.Schema {
 	return map[string]*schema.Schema{
 		"application_id": {
-			Type:        schema.TypeString,
-			Required:    true,
+			Type:     schema.TypeString,
+			Optional: true,
+			ForceNew: true,
+			Deprecated: "This field is deprecated. Policies can now be standalone and reusable by multiple applications." +
```

Comment:
@jacobbednarz can we remove this in v5?

Reply:
yeah, seems reasonable as a pathway to get people into the new stuff but not have it stick around.